kuma/wiki/constants.py

```diff
@@ -222,27 +222,6 @@
     ('inprogress', _('Localization in progress - not completely translated yet.')),
 )
 
-# TODO: This is info derived from urls.py, but unsure how to DRY it
-RESERVED_SLUGS = (
-    r'ckeditor_config\.js$',
-    r'watch-ready-for-review$',
-    r'unwatch-ready-for-review$',
-    r'watch-approved$',
-    r'unwatch-approved$',
-    r'\.json$',
```

Comment:
I removed this from the list of tests because I suspected it was added to check for clashes with  [this url pattern](https://github.com/mozilla/kuma/blob/7ad96618ac0f52f853e6f4d0256e951bbfb40c8f/kuma/wiki/urls.py#L101), but that pattern seemed to only match `docs.json`, so no documents would ever clash with it since they'd always be `docs/...`?

Reply:
👍 Your explanation makes sense. It looks like ``re.match`` is used, so this would only match at the beginning of the slug, and stuff like manifest.json would be allowed.